tests/permissions/permissions.csv

```diff
@@ -53,3 +53,7 @@ comment,,,,,,,test expressions where first item is lambda from .q
 true,,,q,(`a`b`c!1 4 7)~.pm.expr[`tom;"first each d"],,,test expression with each
 fail,,,q,.pm.expr[`harry;"first each d"],,,harry doesn't have access to d
 
+comment,,,,,,,test projection functionality works with both value and eval
+true,,,q,.pm.expr[`tom;({[x;y]y~`ABCD};`x;`ABCD)],,,passed to value and variable used literally
+true,,,q,.pm.expr[`tom;({[x;y]y~`ABCD}[`x];`ABCD)],,,passed to eval and looks up variable
```

Comment:
I think these comments are a little misleading - in particular the second one describes the _old_ (incorrect) behaviour that has been fixed. Projections are no longer going to be passed to eval, so the symbol will be used literally, the same as in the first. In both cases, it's a symbol rather than a variable.

I think something like

```suggestion
true,,,q,.pm.expr[`tom;({[x;y]y~`ABCD};`x;`ABCD)],,,passing a symbol argument to a lambda function
true,,,q,.pm.expr[`tom;({[x;y]y~`ABCD}[`x];`ABCD)],,,passing a symbol argument to a projection
```